events/wc_events/wc_lordaeron_events.txt

```diff
@@ -440,4 +633,884 @@ lordaeron.1103 = {
+				years = 2
+			}
+		}
+		else = { 
+			add_character_modifier = { 
+				modifier = wc_arcanist_equipped_troops
+				years = 2
+			}
+		}
+		remove_short_term_gold = minor_gold_value
+	}
+
+	# reject their help
+	option = { 
+		name = forsaken.9002.opt.c
+	}
+}
+
+# laboratory accident
+forsaken.9003 = { 
+	type = character_event
+	title = forsaken.9003.title
+	theme = battle
+
+	override_background = {
+		reference = burning_building
+	}
+
+	right_portrait = { 
+		character = root
+		animation = worry
+	}
+
+	desc = { 
+		first_valid = { 
+			triggered_desc = { 
+				trigger = { 
+					faith = faith:echo_of_life
+				}
+				desc = forsaken.9003.desc.arcanist # sinister decription, it's our facility
+			}
+			triggered_desc = { 
+				trigger = { 
+					OR = { faith = faith:forsaken_cult faith = faith:alchemical_materialism }
+				}
+				desc = forsaken.9003.desc.alchemist # dispassionate decription, it's our facility
+			}
+			desc = forsaken.9003.desc.other_alchemist # empathetic decription, it's not our facility
+		}
+	}
+
+	trigger = { 
+		NOR = { 
+			var:propaganda_campaign ?= flag:religious
+			var:propaganda_campaign ?= flag:cultural_and_religious
+		}
+		any_held_title = { 
+			tier = tier_county
+			development_level > 2
+			any_province = {
+				has_building_or_higher = temple_01
+			}
+		}
+	}
+
+	immediate = {
+		random_held_title = {
+			limit = { 
+				tier = tier_county
+				development_level > 2
+				any_province = {
+					has_building_or_higher = temple_01
+				}
+			}
+			change_development_progress_with_overflow = -125
+			save_scope_as = affected_county
+		}
+	}
+
+	# rebuild the laboratory
+	option = { 
+		trigger = { 
+			OR = { faith = faith:forsaken_cult faith = faith:alchemical_materialism faith = faith:echo_of_life }
+		}
+		name = forsaken.9003.opt.a
+		remove_short_term_gold = medium_gold_value
+		scope:affected_county = {
+			# placeholder until we add a lab building
+			add_county_modifier = {
+				modifier = wc_medium_development_growth_factor
+				years = 2
+			}
+		}
+	}
+
+	# salvage materials
+	option = { 
+		name = forsaken.9003.opt.b
+		add_gold = minor_gold_value
+		scope:affected_county = {
+			add_county_modifier = {
+				modifier = wc_wreckage_salvaging
+				years = 2
+			}
+		}
+	}
+
+	# cast blame on religious minorities
+	option = {
+		trigger = { 
+			any_held_title = {
+				NOT = { faith = root.faith }
+			}
+			NOR = { 
+				var:propaganda_campaign ?= flag:religious
+				var:propaganda_campaign ?= flag:cultural_and_religious
+			} 
+		}
+		name = forsaken.9003.opt.c
+		
+		if = { 
+			limit = { var:propaganda_campaign ?= flag:cultural }
+			custom_tooltip = { 
+				text = propaganda_campaign_cultural_and_religious_tooltip
+				start_cultural_and_religious_propaganda_campaign = yes
+			}
+		} else = { 
+			custom_tooltip = { 
+				text = propaganda_campaign_religious_tooltip
+				start_religious_propaganda_campaign = yes
+			}
+		}
+	}
+}
+
+# neighbor raids 
+forsaken.9004 = { 
```

Comment:
going to suggest some changes, dont have do to them but maybe make this option have some chance of the raiding party actually defend against you raiding them back. it makes sense if they have good martial to win but its kinda boring if you win everytime

Reply:
honestly i just got bored making this event lol i might just ice this event tbh